src/server/api/lib/crypto.js

```diff
@@ -0,0 +1,36 @@
+import { getConfig } from "./config";
+const crypto = require("crypto");
+
+const secret = getConfig('SESSION_SECRET');
+const algorithm = "aes-256-cbc";
+
+if (!secret) {
+  throw new Error(
+    "The SESSION_SECRET environment variable must be set to use crypto functions!"
+  );
+}
+
+const key = Buffer.concat([Buffer.from(secret)], 32);
+
+const symmetricEncrypt = value => {
+  const iv = crypto.randomBytes(16);
+  const cipher = crypto.createCipheriv(algorithm, key, iv);
+  let encrypted = cipher.update(value);
+  encrypted = Buffer.concat([encrypted, cipher.final()]);
+  return iv.toString('hex') + ':' + encrypted.toString('hex');
+};
+
+const symmetricDecrypt = encrypted => {
+  let parts = encrypted.split(':');
+  const iv = Buffer.from(parts.shift(), 'hex');
```

Comment:
any reason to use hex vs. base64?

Reply:
I think the IV needs to be hex because it needs to stay exactly 16 bytes, but the encrypted text could probably be base64, if there's a reason to prefer that.